src/IntercomResources.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Intercom;
+
+abstract class IntercomResources
+{
+    /**
+     * @var IntercomClient
+     */
+    protected $client;
+
+    /**
+     * IntercomBulk constructor.
```

Comment:
This line would need to be updated to match the class name 😃

As a side note, since PHP7 argument type declarations were introduced, most of the phpdoc comments in this repo could be probably killed 🔪

Reply:
I guess that this makes sense for a project, but that enforcing comments in a package enforces and helps people to contribute.
In any way, if useless comments were to be removed, it could be a part of another PR